spec/requests/reservation_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
```

Comment:
[OPTIONAL] I think it'd be a good idea to change the name of the file to `reservations_controller_spec.rb` for consistency purposes.